lib/js/components/Banner/Banner.vue

```diff
@@ -86,64 +110,73 @@
 	width: 100%;
 
 	&__inner {
+		--ds-banner-title-color: #{$color-neutral-text-strong};
+
 		border-radius: $radius-m;
 		border-style: solid;
 		border-width: 1px;
 		display: flex;
 		flex-direction: column;
+		gap: $space-xs;
 		padding: $space-xs;
 
 		@container (width > 500px) {
-			&.-ds-horizontal {
-				#{$self}__iconContainer {
-					padding: $space-2xs 0;
-				}
-
-				#{$self}__header {
-					padding: 0 $space-2xs;
-				}
+			#{$self}__header {
+				padding: 0 $space-2xs 0 $space-4xs;
+			}
 
-				#{$self}__expander {
-					padding: $space-xs $space-4xs $space-xs 0;
-				}
+			#{$self}__textWrapper {
+				flex-direction: row;
+			}
 
-				#{$self}__textWrapper {
-					flex-direction: row;
-				}
+			#{$self}__rightWrapper {
+				padding: 0 0 0 $space-s;
```

Comment:
this spacing is visible even when right slot is empty. Did you consider migrating it to `gap`?